### PR TITLE
Fix/death animation desync caused by /spawn and setHealth(20) while death

### PR DESCRIPTION
**Pull request by makindotcc**

### CODE REVIEW

**Comment by makindotcc** [comment]
I wanted to create a pull request in our fork 💀

**Comment by makindotcc** [comment]
okay, now i think it is ready to pr yours repo

**Comment by makindotcc** [comment]
This glitch looks like this:
image

and mine patch should this fix this

File: src/main/java/com/planetgallium/kitpvp/command/MainCommand.java [hunk 1]
```diff
@@ -304,21 +310,21 @@ public void run() {
...
                                             XSound.play(p, "BLOCK_NOTE_BLOCK_SNARE, 1, 1");
 
-                                            if (beforeLocation.getBlockX() != p.getLocation().getBlockX() || beforeLocation.getBlockY() != p.getLocation().getBlockY() || beforeLocation.getBlockZ() != p.getLocation().getBlockZ()) {
-
-                                                p.sendMessage(messages.getString("Messages.Error.Moved"));
-                                                spawnUsers.remove(p.getName());
-                                                cancel();
-
-                                            }
-
```
Comment (cervinakuy) [inline src/main/java/com/planetgallium/kitpvp/command/MainCommand.java:321]: Is this even relevant to fixing the death animation?

File: src/main/java/com/planetgallium/kitpvp/Game.java [hunk 2]
```diff
@@ -65,8 +65,10 @@ public void onEnable() {
 		pm.registerEvents(getArena().getKillStreaks(), this);
 		
 		getServer().getMessenger().registerOutgoingPluginChannel(this, "BungeeCord");
-	    getCommand("kitpvp").setExecutor(new MainCommand(this));
-		
+		MainCommand mainCommand = new MainCommand(this);
+		pm.registerEvents(mainCommand, this);
+		getCommand("kitpvp").setExecutor(mainCommand);
```
Comment (cervinakuy) [inline src/main/java/com/planetgallium/kitpvp/Game.java:70]: C

File: pom.xml [hunk 3]
```diff
@@ -96,7 +96,7 @@
         <dependency>
             <groupId>com.sk89q.worldguard</groupId>
             <artifactId>worldguard-bukkit</artifactId>
-            <version>7.0.0</version>
+            <version>7.0.6</version>
```
Comment (cervinakuy) [inline pom.xml:99]: B

File: pom.xml [hunk 4]
```diff
@@ -66,7 +66,7 @@
                         <configuration>
                             <createDependencyReducedPom>false</createDependencyReducedPom>
                             <shadedArtifactAttached>false</shadedArtifactAttached>
-                            <outputFile>/Users/nico/Developer/SpigotL/plugins/${project.artifactId}.jar</outputFile>
+                            <outputFile>target/${project.artifactId}.jar</outputFile>
                         </configuration>
```
Comment (cervinakuy) [inline pom.xml:70]: A

**Review by cervinakuy — CHANGES_REQUESTED** [review]
Some changes (commented A, B, C) appear irrelevant to your overarching goal with the PR (fixing death animation desync). Please create one PR per requested change.
Additionally, while I recognize I don't have any code style guidelines at this time, [e0bdd05](https://github.com/Matt4499/KitPvP/commit/e0bdd05c3bbf7b43226eea080af1245857dc398b) seems like it reformats the entire project code.
Please address these changes at some point in the next weeks, otherwise I will close this

**Comment by makindotcc** [comment]
I don't think I can do any changes anymore, because I'm not working at "kwadratowa" and I don't have access to this repo. In this PR there should be only this commit: [1ec7474](https://github.com/cervinakuy/KitPvP/pull/201/commits/1ec74744d0e424bce029b65f31da6ac20b5cdaa0), but I accidentally created a PR to your repo instead our.

**Comment by makindotcc** [comment]
I'm closing it, because probably I cannot change it anymore.

**Pull request closed without merging**